lintro/ai/fix.py

```
@@ -0,0 +1,629 @@
+        per_cost = response.cost_estimate / count
+        for suggestion in suggestions:
+            suggestion.tool_name = tool_name
+            suggestion.input_tokens = per_input
+            suggestion.output_tokens = per_output
+            suggestion.cost_estimate = per_cost
+
+        logger.debug(
+            f"Batch fix generated {len(suggestions)} suggestions "
+            f"for {file_path} ({len(file_issues)} issues)",
+        )
+        return suggestions
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)